.travis/before_install

```diff
@@ -2,7 +2,7 @@
 
 if [ -z "$OS_TYPE" ]; then
     # default is ubuntu
-    curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | /bin/sh -s -- -b $(go env GOPATH)/bin v1.15.0
+    curl https://raw.githubusercontent.com/golang/dep/master/install.sh | /bin/sh
```

Comment:
sorry, I missed this one before. We don't want to be using dep anymore. Is there something I'm missing?

Reply:
I think it was needed at one point, but apparently not anymore.  I take it out in #3282 and also from another place in the circle ci config.